es-app/src/views/gamelist/BasicGameListView.h

```diff
@@ -11,9 +11,9 @@ class BasicGameListView : public ISimpleGameListView
 	BasicGameListView(Window* window, FileData* root);
 
 	// Called when a FileData* is added, has its metadata changed, or is removed
-	virtual void onFileChanged(FileData* file, FileChangeType change);
```

Comment:
Compiler warning:

In file included from /home/devel/temp/EmulationStation_Stable/es-app/src/views/gamelist/BasicGameLis
tView.h:6:
/home/devel/temp/EmulationStation_Stable/es-app/src/views/gamelist/ISimpleGameListView.h:20:15: warni
ng: 'onFileChanged' overrides a member function but is not marked 'override' [-Winconsistent-missing-
override]
        virtual void onFileChanged(FileData* file, FileChangeType change);
                     ^
/home/devel/temp/EmulationStation_Stable/es-app/src/views/gamelist/IGameListView.h:24:15: note: overr
idden virtual function is here
        virtual void onFileChanged(FileData* file, FileChangeType change) = 0;
                     ^